src/bloggy_frontend/src/pages/Posts.jsx

```diff
@@ -1,41 +1,49 @@
-import React, { useEffect, useState } from 'react'
-import { Container, Row, Col, Card, Button } from 'react-bootstrap'
+import React from 'react'
+import { Container, Row, Col, Card, Button, Spinner } from 'react-bootstrap'
+import { useQuery } from '@tanstack/react-query'
 import { bloggy_backend } from '../../../declarations/bloggy_backend'
```

Comment:
Since we use camelCase everywhere, you can change the import like this 
```suggestion
import { bloggy_backend as bloggyBackend } from '../../../declarations/bloggy_backend'
```